components/shopify-widget/index.tsx

```diff
@@ -57,11 +85,18 @@ const ShopifyWidget = () => {
                 </Text>
               </Flex>
               <PsyButton
-                onClick={() => {
-                  console.log("insert claim logic");
+                // eslint-disable-next-line @typescript-eslint/no-empty-function
+                onClick={() => {}}
+                customStyle={{
+                  width: "100%"
                 }}
+                isDisabled={!userHoldsPOAPToken}
               >
-                Claim Here
+                {address
+                  ? userHoldsPOAPToken
+                    ? "Claim Here"
+                    : "Cannot Claim"
```

Comment:
is this the copy from the design?

Reply:
This change was made based on this interaction
https://linumlabs-workspace.slack.com/archives/C06MT4775FE/p1728033867703989

but I can change the copy if needed?